ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/sno/test/cucushift-installer-rehearse-aws-ipi-sno-test-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+echo "************ single-node test command ************"
+
+# Tests execution
+set +e
+
+NODE_NAME=`oc get node  |grep master | awk '{print $1}'`
+Instance_Type=`aws ec2 describe-instances --filters "Name=private-dns-name,Values=${NODE_NAME}" --query "Reservations[*].Instances[*].[InstanceType]"`
```

Comment:
* the default of aws output in Prow is `json`, so the output of this command would be a _nested-list_, not a _string_
* region param is missing

Reply:
I have added the region parameter.

After defining the **Instance_ Type** variable, I converted it to _string_  in the following sentence.